### PR TITLE
MYC-146 added SuppliedSignature functionality

### PR DESCRIPTION
**Pull request by BtencateSphereon**

### CODE REVIEW

File: src/main/OPBuilder.ts [hunk 1]
```diff
@@ -1,14 +1,23 @@
 import { getUniResolver } from '@sphereon/did-uni-client';
+import {EcdsaSignature} from 'did-jwt/lib/util';
```
Comment (nklomp) [inline src/main/OPBuilder.ts:2]: Formatting

File: src/main/types/JWT.types.ts [hunk 2]
```diff
@@ -1,7 +1,8 @@
+import { EcdsaSignature } from 'did-jwt/lib/util';
 import type { DIDResolutionResult, VerificationMethod } from 'did-resolver';
 // import type {JWK} from "jose/types";
 
-export type Signer = (data: string | Uint8Array) => Promise<string>;
+export type Signer = (data: string | Uint8Array) => Promise<EcdsaSignature | string>;
```
Comment (nklomp) [inline src/main/types/JWT.types.ts:5]: Guess we should add a comment that this is the signer type/interface conforming to the DID-JWT module

**Review by nklomp — APPROVED** [review]
Still the packages/yarn issue. Rest looks okay though

edit: It seems you need to run prettier

**Comment by BtencateSphereon** [comment]
test are failing, looking at it now

**Pull request merged**